src/storage/src/sink/kafka.rs

```diff
@@ -1016,45 +1014,40 @@ where
     // our internal state after the send loop
     let mut progress_update = None;
 
-    let shutdown_flush = Cell::new(false);
-
     // We want exactly one worker to send all the data to the sink topic.
     let hashed_id = id.hashed();
     let is_active_worker = (hashed_id as usize) % scope.peers() == scope.index();
 
     let mut input = builder.new_input(&stream, Exchange::new(move |_| hashed_id));
 
-    builder.build(move |_capabilities| async move {
+    let button = builder.build(move |_capabilities| async move {
```

Comment:
wait i feel like we should explicitly drop these

Reply:
you mean the button?

Reply:
the caps!